README.md

```diff
@@ -83,7 +98,7 @@ use Kirschbaum\Commentions\Filament\Actions\CommentsTableAction;
 ])
 ```
 
-3. Or as a header action:
+Or header actions:
```

Comment:
nit: Could just say "Header actions" so it matches the other headers and reads less like options with "or"